src/components/CreatePost.tsx

```diff
@@ -1,27 +1,59 @@
 import { useUser } from "@clerk/nextjs";
+import AvatarAtom from "./atoms/AvatarAtom";
+import { api } from "~/utils/api";
+import { useState } from "react";
+import toast from "react-hot-toast";
 
 const CreatePost = () => {
   const { user } = useUser(); 
   if (!user) return null;
 
+  const [input, setInput] = useState("")
+
```

Comment:
```suggestion
const onChange = (e: FormEvent) => {
    setInput(e.target.value)
}
```
こんな感じで変更すると
他にもっと`onChange`に対して処理を増やす時に
JSXの中に書いてあるより読みやすくなると思います！
ポイントは、JSX内に関数をかかないようにすることで可読性を担保できます！